middleware/kong/detectors-kong.tf

```diff
@@ -0,0 +1,49 @@
+resource "signalfx_detector" "heartbeat" {
+  name = "${upper(join("", formatlist("[%s]", var.prefixes_slug)))}[${upper(var.environment)}] Kong heartbeat"
```

Comment:
I'd remove the `upper()` function calls to leave more control on the resulting detector name to the user of the modules. I find that especially desired since these are publicly released.

Reply:
it was to preserve / enforce a consistency but ok it makes sens